src/data/num/basic.lean

```diff
@@ -48,17 +50,29 @@ instance : inhabited znum := ⟨0⟩
 
 namespace pos_num
 
+  /--
+  Appending a bit to a `pos_num`, where `bit tt x = x1` and `bit ff x = x0`.
```

Comment:
To my ear, the grammar of a doc string beginning with "Appending..." ("_ing" in general) is a little off. Would you mind rephrasing the ones that start like this? My ideal style would be "`bit b n` appends the bit `b` to `n`, ..." but I won't insist on exactly this!

Reply:
Yeah, I wasn't sure on that one. I think what you're suggesting sounds better.